twenty_forty_eight/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # CHANGELOG - Twenty Forty Eight
 
-1.0.0 / 2021 - 12 - 10
-==================
-### Changes
+## 1.0.0 / 2021 - 12 - 10
```

Comment:
The date here seems weird; is "making the date consistent across changelogs" part of the formatting work happening here?

Reply:
I didn't change the content of the date, in my script I just checked / modified to ensure that it was formatted as a header